detection_rules/custom_rules.py

```diff
@@ -57,7 +57,7 @@ def create_test_config_content() -> str:
 
 @custom_rules.command('setup-config')
 @click.argument('directory', type=Path)
-@click.argument('kibana-version', type=str, default=load_etc_dump('packages.yml')['package']['name'])
+@click.argument('kibana-version', type=str, default=load_etc_dump('packages.yaml')['package']['name'])
```

Comment:
I think it's a bit too much logic and data structure assumptions for the decorator parameters. It would be easier to see if we make `kibana-version` optional and, if it is not set, provide the default value in the method body